clusters/app.ci/ocs/admin_rbac.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Test infrastructure that supports OCS
+    openshift.io/display-name: OCS CI
+  name: ocs
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: ocs
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ocs-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ocs
+  name: image-puller
+---
+kind: Group
+apiVersion: v1
+metadata:
+  name: ocs-admins
+users:
+- vasukulkarni
+- clacroix12
+- petr-balogh
+- vavuthu
+- shylesh
+- zmc
+- andrewschoen
```

Comment:
Vijay, can you please add user `dkhandel` here.

Reply:
Done